Enhancement/live update message

olgakup opened the pull request:
This fixes #403

This is ready for review. PLEASE put in the correct traffic header: * or <http://ethvm.${DOMAIN}>

In This PR: 
- Implemented App Live Update Message
- Removed Vue Icon from tab and updated description
- Updated GitHub link in Footer and  BTC donation address

aldoborrero commented on apps/explorer/src/core/components/ui/AppLiveUpdate.vue:
```diff
@@ -0,0 +1,68 @@
+  lastBlock: string = null
+  update = false
+  newTxs = 1000000
+  /*
```
Add space between variable newTxs and comment

olgakup replied:
Done

aldoborrero commented on apps/explorer/src/core/components/ui/AppLiveUpdate.vue:
```diff
@@ -0,0 +1,68 @@
+
+  beforeDestroy() {
+    this.$eventHub.$off([Events.NEW_BLOCK_METRIC])
+  }
```
Add space between } and comment

olgakup replied:
done

aldoborrero commented on docker-compose.yaml:
```diff
@@ -91,12 +91,10 @@ services:
       - "traefik.enable=true"
       - "traefik.default.protocol=http"
       - "traefik.frontend.rule=Host:api.ethvm.${DOMAIN}"
-      - "traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://ethvm.${DOMAIN}||Access-Control-Allow-Credentials:true"
```
Why is this line removed?

olgakup replied:
Hey i was using it instead of Cors, so i can use Chrome. I will put it back when done :)

olgakup replied:
This is what I am using      
 - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:http://localhost:8080||Access-Control-Allow-Credentials:true'

olgakup replied:
Sorry for gazilion messages, in develop its:  - 'traefik.frontend.headers.customResponseHeaders=Access-Control-Allow-Origin:*||Access-Control-Allow-Credentials:true'
Can you please put the correct one. Not sure which one to use * or http://ethvm.${DOMAIN}

olgakup commented:
This is ready for review.

aldoborrero commented:
@olgakup I think we should take into consideration other factors. For example:

- If, as a user, I'm on PageBlocks.vue and new blocks appears and I click on those I'm successfully refreshing the page and loading newer blocks but I'm not updating the total amount of new blocks that we currently have (so the number of total blocks will be always the same).
- If, as a user, I'm on PageTxs.vue and new txs arrives, happens the same, the total number of Txs is never updated, and also the pagination will be using the old value (the field from).

We need to fix that before merging.

olgakup commented:
@aldoborrero I added total blocks/txs update. I also fixed live component logic implementation.

Let me know if I am missing anything. Otherwise this should be ready.
